src/Add/Add.js

```diff
@@ -9,9 +9,10 @@ import {
   TextField,
   Button,
 } from "@material-ui/core";
-import "./Sort.scss";
+import "./Add.scss";
```

Comment:
imports style

Reply:
i corrected this comment, but it doesn't show up here